catalog/app/components/BucketIcon/__snapshots__/BucketIcon.spec.tsx.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/BucketIcon class names should apply \`\` className if src is set 1`] = `
+<img
+  alt="Custom className"
+  className="makeStyles-root-1 C A"
+  src="https://custom-src"
+/>
+`;
+
+exports[`components/BucketIcon class names should apply \`stub\` className if no src 1`] = `
+<img
+  alt="Stub className"
+  className="makeStyles-root-1 S A"
+  src="IMAGE_MOCK"
+/>
+`;
+
+exports[`components/BucketIcon class names should apply className 1`] = `
+<img
+  alt="Set className"
+  className="makeStyles-root-1 A"
+  src="https://custom-src"
+/>
+`;
+
+exports[`components/BucketIcon should render custom src 1`] = `
+<img
+  alt="Custom src"
+  className="makeStyles-root-1"
+  src="https://custom-src"
+/>
+`;
+
+exports[`components/BucketIcon should render default when no src 1`] = `
+<img
+  alt="No src"
+  className="makeStyles-root-1"
+  src="IMAGE_MOCK"
```

Comment:
why does it render `IMAGE_MOCK` here? what does that mean?

Reply:
It's a mock for bucket.svg (it's a mock for any media file):
it is set here https://github.com/quiltdata/quilt/blob/master/catalog/jest.config.js#L29
and the content of this mock is https://github.com/quiltdata/quilt/blob/master/catalog/internals/mocks/image.js

Reply:
Probably, we can change to something, that mention exported file name, for example, `bucket.svg-mock`